### DISCUSSION TITLE
Make collections (<FeaturedProducts/>) in homepage visible

### DISCUSSION THREAD

**Discussion by gdauster** [medusajs/nextjs-starter-medusa#409] [Q&A]
Hello,
I was not able to display a collection of product in homepage. Not sure if it's an issue or just me.
I've created a collection from Medusa dashboard and inserted 2 items.
But the view in front page is empty. Do i miss something ?

Collections are normally loaded elsewere (footer, dedicated page, ...)

I'm running on v2 branch, downloaded today and a fresh install of Medusa

Regards,
Guillaume

**Comment by oumaru894** [comment]
faced the similar problem. The collections were not rendering on the home page. and mine was only happening in production. It issue was because in my middleware, the cache was not being set. I solve it by using revalidate instead of using the cache which was not goinng with me at the moment.

You shoul also consider add regions in the admin panel
